### DISCUSSION TITLE
Error while trying to access training

### DISCUSSION THREAD

**Discussion by dyefymenko** [uwrobotics/uwrt_mars_rover#163] [Q&A]
Hello! I'm a new member of the team, so I wanted to look through training materials, but I keep getting this error after logging in with my waterloo account: 
![image](https://user-images.githubusercontent.com/84146061/138376152-faaa385f-7520-4cf6-bd56-05a65b10b62b.png)
Does anyone know how to resolve this?

**Comment by wmmc88** [comment]
You shouldnt need to login to see the wiki. All of our confluence stuff is public.

You might want to notify uWaterloo IST about that cause they handle most of the admin stuff of the confluence instance. You can open a ticket with IST.

**Reply by wmmc88** [reply]
okay actually for some reason, our confluence space is no longer accessible without logging in.... looking into it now

**Reply by wmmc88** [reply]
@dyefymenko you should be able to view our confluence again without needing to log in.

**Comment by niiquaye** [comment]
Hello,
Did you try using a different browser, or how about restarting.
